django-server/server/api/serializers.py

```diff
@@ -28,4 +28,21 @@ class CafeteriaSerializer(serializers.ModelSerializer):
     class Meta:
         model = Cafeteria
         fields = ['id', 'name', 'description', 'sub_description', 'longitude', 'latitude',
-                  'logo_url', 'address', 'opened_from', 'opened_to']
+                  'logo_url', 'address', 'opened_from', 'opened_to', 'fixed_menu_options']
```

Comment:
@avan1235 Did I break something here?

Reply:
fix this to the version with no fixed menu options in cafeteria serializer - they are downlaoded separately via nested views and it seems to be better solution to get only base data of all cafeteria and download menu for specified one later